apiserver/pkg/storage/calico/converter.go

```diff
@@ -111,6 +111,13 @@ func convertToAAPI(libcalicoObject runtime.Object) (res runtime.Object) {
 		aapi := &aapi.IPAMConfiguration{}
 		IPAMConfigConverter{}.convertToAAPI(lcg, aapi)
 		return aapi
+	// BlockAffinity works off of the libapi objects since
+	// the v3 client is used for mostly internal operations.
+	case *libapi.BlockAffinity:
```

Comment:
This is a consequence of not writing the v3 client to handle our external API objects. I'm thinking this goes with any follow-up PRs though